### PR TITLE
Dropdowns with multiple select

### LINKED ISSUE #43: Dropdowns with multiple select
It could be nice to have a multi-select dropdown to toggle the display of
- CPM, RCM, and/or GCM
- Past performance and/or future change

### LINKED ISSUE #54: Fix sonarcloud code quality check
Sonarcloud is complaining about the [duplication of codes](https://sonarcloud.io/component_measures?metric=new_duplicated_lines_density&selected=eucp-project_data-catalogue%3Acomponents%2Fcpm&view=list&id=eucp-project_data-catalogue).

The duplication is due to the temporal solution related to the display of maps in cpm page. The problem is will be handled when implementing new dropdown multiple select feature and the re-arrangement of tabs.

### PR DESCRIPTION
**Pull request by geek-yang**
Add dropdowns with multiple select feature. We use the external plugin [vue-multiselect](https://www.npmjs.com/package/vue-multiselect).

The "past performance" tab and "future change" tab are merged into one tab.

The single select dropdown component will be replaced by this plugin as well (not that it is not deleted since "Analyse" tab still uses it. It can be removed later).

This PR closes #43 and fixes #54.

### CODE REVIEW

**Comment by geek-yang** [comment]
This PR is ready for review. Please focus on the functionality of multiple dropdown, which allows the user to play with the display of cpm/rcm/gcm maps and past performance/future change experiments.

The styling of the page is not a primary concern in this PR. Given that this PR involves lots of changes already, it is better to address it in another PR, or together with the re-design of the whole cpm page.

**Review by bvreede — APPROVED** [review]
Approving bc gh is being a *****

**Comment by bvreede** [comment]
The name "performance" works, but I would argue for the name "Predictions" instead (or "performance & predictions", but that might be too long), as the main outcome of interest (? at least for me) from these models is their predictions for future change. Having the performance there is valuable, but it is not why the models were developed.

**Comment by Peter9192** [comment]
Is it possible to merge main into this branch to get rid of all the figure changes?

**Comment by geek-yang** [comment]
> Is it possible to merge main into this branch to get rid of all the figure changes?

Good point! I thought I already did, but that was before we merged the PR that changed all the figure names. Thanks for suggesting!

File: components/cpm/Performance.vue [hunk 1]
```diff
@@ -1,54 +1,119 @@
...
+          :close-on-select="false"
+          :clear-on-select="false"
+          :preserve-search="true"
+          :show-labels="true"
+          placeholder="Choose modelling system(s)"
+          label="name"
+          track-by="name"
+          :preselect-first="true"
+        >
+          <template slot="selection" slot-scope="{ values, isOpen }"><span v-if="values.length &amp;&amp; !isOpen" class="multiselect__single">{{ values.length }} modelling system(s) selected</span></template>
```
Comment (Peter9192) [inline components/cpm/Performance.vue:57]: Modelling systems is not clear and also not really accurate. I think I'd prefer either:

- Projects (CMIP, CORDEX, CPM)  (this is accurate)
- Type (global, regional, convection-permitting)   (this holds the middle)
- Resolution (low, medium, high)   (this is easier to understand)

File: components/Navigation.vue [hunk 2]
```diff
@@ -44,7 +44,7 @@ export default {
           title: 'CPM',
           url: '/cpm',
           children: [{ title: 'Explore', url: '/cpm/explore' },
-            { title: 'Past performance', url: '/cpm/performance' },
+            { title: 'Performance', url: '/cpm/performance' },
```
Comment (Peter9192) [inline components/Navigation.vue:47]: I think the page with past performance/future change should be called "explore". The page that's now called 
"explore" can be changed to "Lines of evidence" or so.

**Review by Peter9192 — APPROVED** [review]
Purely focusing on the functionality, I tested this and can confirm it works. The layout and also the code can be improved, but let's do that in follow-up PRs. 

Can we split the dropdowns and figure display into separate components?

Please verify that the live site also works fine after merging this PR

File: components/Navigation.vue (hunk 2 above)
Comment (geek-yang) [reply components/Navigation.vue]: The tabs are renamed accordingly 😃.

File: components/cpm/Performance.vue (hunk 1 above)
Comment (geek-yang) [reply components/cpm/Performance.vue]: Thanks for your suggestions. I prefer the accurate one, the Projects (CMIP, CORDEX, CPM). They are implemented now.

**Comment by geek-yang** [comment]
> Purely focusing on the functionality, I tested this and can confirm it works. The layout and also the code can be improved, but let's do that in follow-up PRs.
> 

@Peter9192 Thanks for your feedback! Just finish addressing your comments. I will go ahead and merge it.

> Can we split the dropdowns and figure display into separate components?
> 

I thought about it before. But since the figure display is only an div with a background figure, it feels a bit over-engineering to wrap it up as a single component. I would suggest we leave it like this and get back to it when we want to polish the code for the whole site, then we can have a better vision about which part could be reused (and make it a new component).


> Please verify that the live site also works fine after merging this PR
> 

No problem. I will keep an eye on it.

**Pull request merged**